_includes/head.html

```diff
@@ -30,7 +30,7 @@
 {%- endif -%}
 <head>
   <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
-  {%- if page.sitemap == false or site.GH_ENV == "gh_pages" %}
+  {%- if jekyll.environment != 'production' %}
```

Comment:
We probably need to keep the `page.sitemap` as well; it's used for pages that are not removed, but no longer present in the TOC (and/or shouldn't be indexed, such as pages about deprecated things)

Reply:
Oh yes indeed, fixed.